config/initializers/alma.rb

```diff
@@ -1,12 +1,13 @@
 # frozen_string_literal: true
 
+ENV["ALMA_API_KEY"] ||= Rails.configuration.alma[:apikey]
+ENV["ALMA_DELIVERY_DOMAIN"] ||= Rails.configuration.alma[:delivery_domain]
+ENV["ALMA_INSTITUTION_CODE"] ||= Rails.configuration.alma[:institution_code]
+ENV["ALMA_AUTH_SECRET"] ||= Rails.configuration.alma[:auth_secret]
+
 Alma.configure do |config|
   # You have to set te apikey
-  config.apikey = Rails.configuration.alma[:apikey]
+  config.apikey = ENV["ALMA_API_KEY"]
```

Comment:
```suggestion
  config.apikey = ENV["ALMA_API_KEY"] ||= Rails.configuration.alma[:apikey]
```

The primo file still has the config as backup.  Should we do that here too?

Reply:
@sensei100 we are already doing that on line 3 above.  So we do not need to redo it again.

Reply:
Ok for me to commit this? I think it is more readable and prevents an inattentive reorder breaking things.

Reply:
I reverted changes to this fie.